### ISSUE TITLE
unable to build

### ISSUE THREAD

**Issue by cardoe** [jameysharp/corrode#102]
I've got ghc 7.10.3 and cabal 1.22 on Ubuntu 16.04. I did the following steps

```bash
export PATH=$HOME/.cabal/bin:$PATH
cabal update
cabal install happy
cabal install alex
cd my-corrode-checkout
cabal install
```

The last steps fails with the following:

```
Registering language-c-0.5.0...
Installed language-c-0.5.0
Configuring corrode-0.1.0.0...
Building corrode-0.1.0.0...
Preprocessing library corrode-0.1.0.0...
markdown-unlit: src/Language/Rust/Corrode/C.lhs: hGetContents: invalid argument (invalid byte sequence)
Failed to install corrode-0.1.0.0
cabal: Error: some packages failed to install:
corrode-0.1.0.0 failed during the building phase. The exception was:
ExitFailure 1
```

I don't have any experience with Haskell so I'm not sure where to go from here.

**Comment by colindean** [comment]
I just successfully installed it following those instructions. I'm not sure what went wrong, other than perhaps you have an unstable checkout. `git pull` on your corrode checkout?

**Comment by colindean** [comment]
#101 seems to be intended to address this.

Oh the things I'm learning as I'm poking around!

**Comment by cardoe** [comment]
You are right. Thank you. This is a duplicate of #96 and fixed by #101 or by setting the locale correctly.
